### ISSUE TITLE
 connect schema servers to morphic.bio domain and verify access

### ISSUE THREAD

**Issue by amnonkhen** [morphic-bio/data-ingestion-development#33]
dev - dev.schema.morphic.bio
prod - schema.morphic .bio
 work on it with Caty

Domains are managed using Route53.

**Comment by amnonkhen** [comment]
There is a problem now that the cloudfront distribution is not visible when trying to create an A record Alias in Route53.

<img width="512" alt="image" src="https://github.com/morphic-bio/data-ingestion-development/assets/927145/a55768c4-69e2-4211-b4ed-2c99f468e9b9">

**Comment by KociOrges** [comment]
From my investigation so far, the CloudFront distribution is not visible when creating a record Alias on Route 53, because the custom domain name (e.g., dev.schema.morphic.bio) which is used for routing traffic, needs to be added as an alternate domain name (CNAME) into that CloudFront distribution. To do this, however, a valid certificate must be first issued on that domain. To issue the certificate, there seems to be some ownership-dependent configuration required, particularly if the domain was not registered through AWS. Thus, need to resolve ownership configuration issues.

**Comment by ESapenaVentura** [comment]
Waiting on Cathy to answer @KociOrges

**Comment by KociOrges** [comment]
The subdomain routing management through Route 53 has now been resolved and the steps for setting up subdomains for MorPhiC are enumerated here: https://docs.google.com/document/d/1_L-c_PLWq-3DBGu_fjpN27RIo_WCbBJUKRvofzomv70/edit

**Comment by gabsie** [comment]
closing this one, as covered in another ticket
